test/kernelabstractions.jl

```diff
@@ -1,8 +1,12 @@
-using KernelAbstractions
+using KernelAbstractions    
+import Parameters: @with_kw, @unpack
```

Comment:
I don't think we have Parameters as dependency in our tests (package dependency and test dependency aren't the same, as you may want to test against another package but the code doesn't actually depend on it). This may work in your local environment, but this may fail at CI. Note in general (since Julia v1.7) can also unpack like this
```julia
(;lmax,mmax) = m.spectral_transform
```
or always rename locally if you don't feel lazy
```julia
maximum_degree_l = m.spectral_transform.lmax
```

Reply:
We can just use the first style so, 
```
(;lmax,mmax) = m.spectral_transform
```

I'd say, I can add this later the `kernelabstractions_adtest` branch